src/tyr/locate_serializer.cc

```diff
@@ -16,6 +62,50 @@ json::ArrayPtr serialize_edges(const PathLocation& location, GraphReader& reader
       auto edge_info = tile->edgeinfo(directed_edge->edgeinfo_offset());
       // they want MOAR!
       if (verbose) {
+        auto live_speed = json::map({});
+        const volatile auto& traffic = tile->trafficspeed(directed_edge);
+        if (traffic.valid()) {
+          live_speed->emplace("overall_speed", static_cast<uint64_t>(traffic.get_overall_speed()));
+          auto speed = static_cast<uint64_t>(traffic.get_speed(0));
+          if (speed == UNKNOWN_TRAFFIC_SPEED_KPH)
+            live_speed->emplace("speed_0", nullptr);
+          else
+            live_speed->emplace("speed_0", speed);
+          auto congestion = (traffic.congestion2 - 1.0) / 62.0;
```

Comment:
Hmm I don't like that there is a transformation here that depends on implementation details from traffictiles.h. Could it be move to that file?

Reply:
Also, isn't it wrong with `traffic.congestion2`?